tests/sentry/issues/test_escalating.py

```diff
@@ -0,0 +1,81 @@
+from uuid import uuid4
+
+import pytest
+
+from sentry.eventstore.models import Event
+from sentry.issues.escalating import query_groups_past_counts
+from sentry.models import Group
+from sentry.testutils import TestCase
+from sentry.testutils.factories import Factories
+from sentry.testutils.helpers.datetime import before_now
+from sentry.utils.snuba import SnubaError, to_start_of_hour
+
+
+class HistoricGroupCounts(TestCase):  # type: ignore
+    def setUp(self) -> None:
+        super().setUp()
+
+    def _load_event_for_group(
+        self,
+        minutes_ago: int = 2,
+        fingerprint: str = "foo-1",
+    ) -> Event:
+        """Creates a new event for a group. It creates one if missing.
+        Use fingerprint to create different groups.
+        """
+        return Factories.store_event(
+            project_id=self.project.id,
+            data={
+                "event_id": uuid4().hex,
+                "message": "some message",
+                # XXX: Let's create a method that help us control which hour this will fall under
+                # XXX: Write tests for this
+                "timestamp": before_now(minutes=minutes_ago).timestamp(),
```

Comment:
Right now, I'm using `60` and `120` minutes in the tests in order to make sure that events fall under the same hour.